projects/aca-playwright-shared/src/api/nodes-api.ts

```diff
@@ -418,6 +418,26 @@ export class NodesApi {
     }
   }
 
+  async getFileVersionType(nodeId: string): Promise<string> {
+    try {
+      const prop = await this.getNodeProperty(nodeId, 'cm:versionType');
+      return prop || '';
+    } catch (error) {
+      console.error(`${this.constructor.name} ${this.getFileVersionType.name}`, error);
+      return '';
+    }
+  }
+
+  async getFileVersionLabel(nodeId: string): Promise<string> {
```

Comment:
The same goes here

Reply:
I change this code with getNodeProperty, removed  getFileVersionLabel method